change-beta/@azure-communication-react-45da9f59-d5f0-420c-a677-f6df8aa71cfd.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "area": "improvement",
+  "workstream": "",
+  "comment": "Port saving display name to local storage from CallwithChat Sample to Chat Sample",
```

Comment:
We probablydon't need the changelogs here as the changes aren't in communication-react package